### PR TITLE
Update branding: New Logo

### PR DESCRIPTION
**Pull request by ZekeZDev**
This is a merge request related to #1 this is the current production version of my design.

This pull request also modifies the README accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
 This is a **fork** of the MultiMC Launcher and not endorsed by MultiMC. The PolyMC community felt that the maintainer was not acting in the spirit of Free Software so this fork was made. Read "[Why was this fork made?](https://github.com/PolyMC/PolyMC/wiki/FAQ)" on the wiki for more details.
 
 ## Packages
-Several source build packages are available, along with experimental pre-built generic packages.
+<a href='https://flathub.org/apps/details/org.polymc.PolyMC'><img width='240' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a>
 
-- An [AUR package](https://aur.archlinux.org/packages/polymc-git/) is available.
-- A Gentoo ebuild is available in the [swirl](https://git.swurl.xyz/swirl/ebuilds) overlay, named `games-action/polymc`. Check the README for instructions on how to add the overlay.
-- A Flatpak is available in [Flathub](https://flathub.org/apps/details/org.polymc.PolyMC). You can install it from there or build it yourself using [this source](https://github.com/flathub/org.polymc.PolyMC).
+Several source build packages are available, along with experimental pre-built generic packages.
```
Comment (ghost) [inline README.md:11]: put this before flathub

File: README.md [hunk 2]
```diff
@@ -50,11 +50,11 @@ Feel free to create an issue if you need help. However, you might find it easier
 
 [![PolyMC Discord](https://img.shields.io/discord/923671181020766230?label=PolyMC%20Discord)](https://discord.gg/xq7fxrgtMP)
 
-For people who don't want to use Discord, we have a Matrix Space which is bridged to the Discord server. Be sure to enable spaces first (Settings -> Labs -> Spaces), and then you may join the space:
```
Comment (ghost) [inline README.md:54]: Why was this removed?

File: README.md [hunk 3]
```diff
@@ -50,11 +50,11 @@ Feel free to create an issue if you need help. However, you might find it easier
...
+For people who don't want to use Discord, we have a Matrix Space which is bridged to the Discord server.
 
 [![PolyMC Space](https://img.shields.io/matrix/polymc:polymc.org?label=PolyMC%20Space&server_fqdn=matrix.polymc.org)](https://matrix.to/#/#polymc:polymc.org)
 
-Matrix's support for spaces is still in development, so if you have issues accessing rooms via the space, then you can join the rooms directly:
+If there is any issues with the space or you are using a client that does not support the feature here are the individual rooms:
```
Comment (ghost) [inline README.md]: ditto

File: program_info/org.polymc.PolyMC.Source.svg [hunk 4]
```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (ghost) [inline program_info/org.polymc.PolyMC.Source.svg:1]: What's this file for?

File: program_info/org.polymc.PolyMC.svg [hunk 5]
```diff
@@ -1,9 +1,21 @@
...
+  <path d="m3.561 16.016s0-3.5642 4.9056-3.5642c4.9069 0 4.9056 3.5642 4.9056 3.5642z" fill="#765338"/>
+  <path d="m8.4667 12.452-4.9056 3.5642-3.0319-9.3311z" fill="#b7835a"/>
+  <path d="m8.4667 12.452 7.9375-5.7669-3.0319 9.3311z" fill="#5b422d"/>
+  <path d="m8.8308 12.716-0.36417 0.26458-0.36417-0.26458c0-0.26458 0.36417-0.26458 0.36417-0.26458s0.36417 0 0.36417 0.26458z" fill="#72b147"/>
+  <path d="m8.4667 12.452s-2e-7 -5.7669 7.9375-5.7669l-0.22507 0.69269-0.91853 1.1965-0.91853 0.13819-0.91853 1.1965-0.91853 0.13819-0.91853 1.1965-0.91853 0.13819-0.91853 1.1965-0.91853 0.13819z" fill="#5a9a30"/>
+  <path d="m8.1025 12.716-0.91853-0.13819-0.91853-1.1965-0.91853-0.13819-0.91853-1.1965-0.91853-0.13819-0.91853-1.1965-0.91853-0.13819-0.91853-1.1965-0.22507-0.69269c7.9375 1e-7 7.9375 5.7669 7.9375 5.7669z" fill="#88b858"/>
+  <path d="m0.52917 6.6846 7.9375 5.7669 7.9375-5.7669-7.9375-5.7669z" fill="url(#linearGradient84726)"/>
+ </g>
+ <path d="m0.75424 7.3773-0.22507-0.69269 7.9375 5.7669 7.9375-5.7669-0.22507 0.69269-7.7124 5.6034z" fill-opacity="0"/>
+</svg>
```
Comment (ghost) [inline program_info/org.polymc.PolyMC.svg:21]: Additionally, there's an extra polymc-logo file

File: README.md [hunk 6]
```diff
@@ -1,20 +1,18 @@
 <p align="center">
-  <img src="https://avatars.githubusercontent.com/u/96310119" alt="PolyMC logo"/>
+  <img src="/program_info/polymc-logo.png#gh-light-mode-only" alt="PolyMC logo"/>
```
Comment (ghost) [inline README.md]: Why not store this in githubusercontent, or on a separate repo?

**Review by ghost — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ZekeZDev) [reply README.md:54]: Not needed anymore its enabled by default, its out of beta.

File: README.md (hunk 6 above)
Comment (ZekeZDev) [reply README.md]: I wasn't sure how we where storing icons etc the program_info seemed to say thats where we would store branding. I am mistaken.

File: program_info/org.polymc.PolyMC.Source.svg (hunk 4 above)
Comment (ZekeZDev) [reply program_info/org.polymc.PolyMC.Source.svg:1]: its the source file that has all of the guides etc used to create the logo. its an unoptimised svg

File: program_info/org.polymc.PolyMC.svg (hunk 5 above)
Comment (ZekeZDev) [reply program_info/org.polymc.PolyMC.svg:21]: theres polymc-logo (light text) and polymc_logo (dark text)

Comment (ghost) [reply program_info/org.polymc.PolyMC.svg:21]: Make it polymc-dark.png and polymc-light.png

File: README.md (hunk 6 above)
Comment (ghost) [reply README.md]: Ok, move it to polymc-{dark,light}.png and it's fine to keep it in program_info

File: README.md (hunk 3 above)
Comment (ZekeZDev) [reply README.md]: Matrix spaces is no longer under development its out of beta and a full feature

Comment (ZekeZDev) [reply README.md]: Some clients however dont support it and makes it more clear

Comment (ghost) [reply README.md]: Ok, switch that "is" to "are" and I can squash merge.

File: program_info/org.polymc.PolyMC.svg (hunk 5 above)
Comment (ZekeZDev) [reply program_info/org.polymc.PolyMC.svg:21]: Fixed They are both named accordingly now
